src/Arcus.Security.Tests.Unit/Core/IHostBuilderExtensionsTests.cs

```diff
@@ -138,5 +138,28 @@ public async Task ConfigureSecretStore_AddEnvironmentVariables_UsesEnvironmentVa
                 Assert.Equal(secretValue, await provider.GetRawSecretAsync(secretKey));
             }
         }
+
+        [Fact]
+        public async Task ConfigureSecretStore_AddEnvironmentVariablesWithPrefix_UsesEnvironmentVariableSecrets()
```

Comment:
Can we add a test where it's not aligned with the prefix to verify it's ignored?

Reply:
Yes, added a test when the secret is not found.